lib/roo_on_rails/rack/valid_identity_service_prefixes.yml

```diff
@@ -0,0 +1,6 @@
+production:
+  - https://deliveroo.co.uk/identity-keys/
+  - https://identity.deliveroo.net/identity/keys/
+staging:
+  - https://test.deliveroo.co.uk/identity-keys/
+  - https://identity-staging.deliveroo.net/identity-keys/
```

Comment:
Can this list be overwritten? Especially because `roo_on_rails` is opensource people might want to overwrite it in their application?

Reply:
They could, but they can't overwrite it in our services, which is the only place that matters to us :)